std_msgs/Header header
float32 tacho_value
float32 v_in

# custom rotations field added here because only the tacho_value is needed for computing it
float32 rotations